go/saltpack/nonce.go

```diff
@@ -10,50 +9,30 @@ import (
 // counter values, and some of which can be random-ish values.
 type Nonce [24]byte
 
-// NonceType is different for the different type of usages of the nonce.
-type NonceType int
-
-const (
-	// NonceTypeEncryptionKeyBox is used for the header of an encrypted message
-	NonceTypeEncryptionKeyBox NonceType = 0
-
-	// NonceTypeEncryptionPayloadBox is used for the payload of an encrypted message
-	NonceTypeEncryptionPayloadBox NonceType = 1
-)
-
-// ForPayloadBox formats this nonce for the ith block of payload.
-func (n *Nonce) ForPayloadBox(i encryptionBlockNumber) *Nonce {
-	n.mutate(uint64(NonceTypeEncryptionPayloadBox) + uint64(i))
-	return n
+func nonceForSenderKeySecretBox() *Nonce {
+	var n Nonce
+	copy(n[:], "saltpack_sender_key\x00\x00\x00\x00\x00")
```

Comment:
Is there a way to make this a constant of type `Nonce`, instead of requiring a function? Go complained when I tried.